### ISSUE TITLE
Named targets

### ISSUE THREAD

**Issue by Kakadu** [ocaml/dune#3309]
It seems that we have an ability to have named dependecies like `(deps (:first-dep foo.cppo.ml) <other files that foo.ml includes>)`. Could we also have named targets? Or at least a variable that expands to first target (like in Makefile)?

**Comment by diml** [comment]
Isn't that already supported?

**Comment by Kakadu** [comment]
I asked at discord about it and rgrinberg  said "We don't have this but there's no reason why we can't add it".

**Comment by diml** [comment]
Right, we might have talked about it but never implemented it. I checked and this is indeed not supported.

**Comment by create2000** [comment]
Hello @gridbugs can i have this assigned to me?
